PluginsAndFeatures/azure-toolkit-for-intellij/rider/resources/messages/RiderAzureMessages.properties

```diff
@@ -547,6 +547,8 @@ inspection.function_app.azurefunctionsversion.version_unsupported=Unsupported Az
 inspection.function_app.azurefunctionsversion.quickfix.set_version=Set Azure Functions version ''{0}''
 
 # Gutter Mark
+gutter.function_app.run.all=Run all functions...
+gutter.function_app.debug.all=Debug all functions...
```

Comment:
_(Minor)_ [IntelliJ UI Guidelines ask us](https://jetbrains.github.io/ui/text/punctuation/) to use the ellipsis character (`…`) in UI instead of three dots `...`